stdlib/public/stubs/SwiftNativeNSXXXBase.mm.gyb

```diff
@@ -117,6 +133,37 @@ extern "C" int32_t swift_stdlib_compareNSStringDeterministicUnicodeCollation(
 }
 
 SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" int32_t
+swift_stdlib_compareNSStringDeterministicUnicodeCollationStack(
+    void *baseAddressLhs, uintptr_t countAndFlagsLhs, void *ownerLhs,
+    void *baseAddressRhs, uintptr_t countAndFlagsRhs, void *ownerRhs) {
+
+  _SwiftStackAllocatedNSContiguousString stackAllocatedStringLhs = {
+      {
+          &_TMCs19_NSContiguousString, // isa.
+      },
+      baseAddressLhs,
+      countAndFlagsLhs, // core.
+      ownerLhs
+  };
+  _SwiftStackAllocatedNSContiguousString stackAllocatedStringRhs = {
+      {
+          &_TMCs19_NSContiguousString, // isa.
+      },
+      baseAddressRhs,
+      countAndFlagsRhs, // core.
+      ownerRhs
```

Comment:
Running clang-format in a .cpp file on this function results in:

```c++
extern "C" int32_t
swift_stdlib_compareNSStringDeterministicUnicodeCollationStack(
    void *baseAddressLhs, uintptr_t countAndFlagsLhs, void *ownerLhs,
    void *baseAddressRhs, uintptr_t countAndFlagsRhs, void *ownerRhs) {

  _SwiftStackAllocatedNSContiguousString stackAllocatedStringLhs = {
      {
          &_TMCs19_NSContiguousString, // isa.
      },
      baseAddressLhs,
      countAndFlagsLhs, // core.
      ownerLhs};
  _SwiftStackAllocatedNSContiguousString stackAllocatedStringRhs = {
      {
          &_TMCs19_NSContiguousString, // isa.
      },
      baseAddressRhs,
      countAndFlagsRhs, // core.
      ownerRhs};
  NSString *lhs = (NSString *)&stackAllocatedStringLhs;
  NSString *rhs = (NSString *)&stackAllocatedStringRhs;

  // 'kCFCompareNonliteral' actually means "normalize to NFD".
  int Result = CFStringCompare((__bridge CFStringRef)lhs,
                               (__bridge CFStringRef)rhs, kCFCompareNonliteral);
  return Result;
}
```

Yes it does put the closing bracket on the same line. I took the liberty to move it because I found this ugly.

I would be happy to format that in any way it pleases though ;).